processor/src/main/java/org/mapstruct/ap/internal/model/assignment/AssignmentWrapper.java

```diff
@@ -81,8 +82,26 @@ public AssignmentType getType() {
         return decoratedAssignment.getType();
     }
 
-     @Override
+    @Override
     public boolean isUpdateMethod() {
         return decoratedAssignment.isUpdateMethod();
     }
+
+    public String getAfterWriteVariable() {
```

Comment:
@sjaakd this and the `getFinishWriteStatement()` are used to return the way we need to start/finish an assignment. If you have some ideas I would be glad to hear them. Maybe you have something better in your refactorings.